build.sbt

```diff
@@ -122,7 +126,8 @@ lazy val cliPackSettings = Def.settings(
       "auxlib",
       "scalalib"
     ).map { lib =>
-      val nativeBinVersion = ScalaNativeCrossVersion.binaryVersion(snVer)
+      val nativeBinVersion =
```

Comment:
We should also add Scala 3 specific dependencies in a similar manner - we would check for binary version of Scala - if it is equal to `3` to should also add dependency to `scala3lib`. Also in case of `scalalib` we should check if it has the correct binary version - `scalalib` would be only published for 2.13. 
I guess we should define additional value eg. `scalaStdLibraryModuleIDs` where we would resolve correct artifact names

Reply:
Thank you for finding this. I might need to add a similar thing to `scala-cli` (and maybe `bloop` possibly) - I completely forgot about the added dependencies. `scala3lib` is the only one for Scala 3 right?

Reply:
Yes, it's the only new dependency that is needed by Scala 3. When it comes to `scala-cli` `scala3lib` depends on `scalalib` so I guess coursier should handle transitive dependencies.